docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,214 @@
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 4]
+    :end-before: # [/example 4]
+
+.. image:: images/lighting-setups-images/5.jpg
+    :width: 20em
+
+`Multiple Light Setups`_
+========================
+
+In some cases (such as when emulating room-level lighting for scenes with multiple rooms), one light setup may not be sufficient to achieve a desired visual result.
+To use multiple custom lighting setups at the same time, simply register a name after creation.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 5]
+    :end-before: # [/example 5]
+
+To use a specific light setup for a new object, pass in the name as a parameter to :ref:`Simulator.add_object`.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 6]
+    :end-before: # [/example 6]
+
+.. image:: images/lighting-setups-images/6.jpg
+    :width: 20em
+
+Retrieve a copy of an existing configuration with :ref:`Simulator.get_light_setup` to query light properties, make small changes, or reduce the effort of creating a new light setup from scratch.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 7]
+    :end-before: # [/example 7]
+
+Updates to existing light setups will update all objects using that setup, allowing control over groups of objects with consistent lighting.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 8]
+    :end-before: # [/example 8]
+
+.. image:: images/lighting-setups-images/7.jpg
+    :width: 20em
+
+Some situations may warrant a change in the lighting setup for a specific object already instanced into the scene (such as when moving an object to a new room).
+The light setup any individual object uses can be changed at any time with :ref:`Simulator.set_object_light_setup`.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 9]
+    :end-before: # [/example 9]
+
+.. image:: images/lighting-setups-images/8.jpg
+    :width: 20em
+
+`Feature Detail Review`_
+========================
+
+A light setup consists of a set of :ref:`LightInfo` structures defining the common configuration
+of a set of point lights used to render objects in a scene. Once defined and registered,
+a light setup can be assigned to any subset of objects in the scene, including the scene asset itself.
+
+Each :ref:`LightInfo` structure in a light setup defines the `color`, `position`, and :ref:`LightPositionModel` of a single point light source.
+The :ref:`LightPositionModel` defines the coordinate frame of the light.
+
+Each light setup is registered in the simulator via a unique key. Two default lighting setups are pre-defined:
+
+
+- :ref:`DEFAULT_LIGHTING_KEY`
+
+  8 white, GLOBAL lights at scene bounding box corners.
+
+- :ref:`NO_LIGHT_KEY`
```

Comment:
(I will do a more detailed pass over this tomorrow, but this caught my attention) I *think* it won't work like this (will produce a warning on the output and render as monospace, not a link). Instead you'll need to say the whole path, you can still make the title shorter tho):

```suggestion
- :ref:`NO_LIGHT_KEY <habitat_sim.gfx.NO_LIGHT_KEY>`
```

Same I guess for all others -- if a `:ref:` doesn't render a link, there's something off.

Reply:
I added ref prefixes to address this. It works on my build 🤔 
```
:ref-prefix:
    habitat_sim.gfx
    habitat_sim.simulator
    habitat_sim.sim
    habitat_sim.agent
    habitat_sim.attributes
```

Reply:
Ah, sorry about that, missed this part (it's too late here :sweat_smile:).

Tho to avoid confusing users I would use those in moderation, similarly to best practices with `using namespace`. One rule could be to use `:ref-prefix:` only if the tutorial code in question does the equivalent (`from habitat_sim.sim import Simulator` for example). I'm trying to follow this rule closely in Magnum docs -- always specifying the fully qualified path in all links (`Shaders::Phong` for example) unless it's undoubtedly clear from the context.